hazelcast/src/main/java/com/hazelcast/replicatedmap/impl/ReplicatedMapService.java

```diff
@@ -333,6 +337,7 @@ private Map executeClearOnMembers(Collection<Address> members, String name) {
             }
             Operation operation = new ClearLocalOperation(name);
             InvocationBuilder ib = operationService.createInvocationBuilder(SERVICE_NAME, operation, address);
+            ib.setTryCount(1);
```

Comment:
can you please comment on why this is `1`? And can you make it a static final constant?